### DISCUSSION TITLE
code discussions.

### DISCUSSION THREAD

**Discussion by bishtsurajsoftage** [bishtsurajsoftage/random#11] [General]
writing another discussion for our nodejs app .
Editing while in the discussion.

**Comment by adisoftage** [comment]
HIHHI

**Comment by bishtsurajsoftage** [comment]
I am writing this comment to ensure the code runs successfully in production.

**Comment by adisoftage** [comment]
**Bugs in the file:**

- Isn't Optimized
- Can't be opened on iOs
- Images are laggy when opened

**Reply by bishtsurajsoftage** [reply]
I 'll try to remove the bugs as soon as possible.

**Comment by adisoftage** [comment]
**immediately stop the app from deployment !!**
